src/Util/Http.php

```diff
@@ -46,14 +43,10 @@
      *
      * @return array the parsed headers
      *
-     * @throws \Exception Thrown if unable to parse the headers
+     * @throws Exception Thrown if unable to parse the headers
      */
-    public static function parseHeaders($rawHeaders)
+    public static function parseHeaders(string $rawHeaders) : array
     {
-        if (!is_string($rawHeaders) || trim($rawHeaders) === '') {
```

Comment:
you should probably keep the `trim($rawheaders)` check

Reply:
done